types/fs-extra/index.d.ts

```diff
@@ -137,20 +306,20 @@ export interface CopyOptions {
 }
 
 export interface MoveOptions {
-    clobber? : boolean;
+    clobber?: boolean;
```

Comment:
`clobber` was renamed to `overwrite` in v2 (clobber technically still works, but it's undocumented and will be removed)

Also fix this in `CopyOptions`